components/Collections/src/MDCCollectionInfoBarView.h

```diff
@@ -149,14 +156,10 @@ typedef NS_ENUM(NSUInteger, MDCCollectionInfoBarViewStyle) {
 @property(nonatomic, assign) BOOL allowsTap;
 
 /** The optional style that the info bar can be shown. */
-@property(nonatomic, assign) MDCCollectionInfoBarViewStyle style;
+@property(nonatomic, assign) MDCInfoBarStyle style;
```

Comment:
Yes it can be changed anytime. But to help clarify, I updated the comment for `MDCInfoBarKind kind` to make sure it is known that can only change at `init`.